### DISCUSSION TITLE
GitHub is looking for your top use cases for project templates!

### DISCUSSION THREAD

**Discussion by brontevdh** [community/community#35253] [Projects and Issues] [answered]
Hi :wave:

The Planning and Tracking team at GitHub are keen to build features to support your productivity! We’re looking to expand project template functionality and if you’d like to share your thoughts, we’d love to hear from you. Please share with us - [in this quick survey](https://github.surveymonkey.com/r/WTW7BMJ) - why project templates are important to you and your top use case(s).

![image](https://media1.giphy.com/media/3oKIPdiPGxPI7Dze7u/giphy-downsized.gif?cid=6104955esylxpalmk7ap2fw2tqp5gks2jh2qjnb7uhhisjno&rid=giphy-downsized.gif&ct=g)

_Many thanks!_

**Comment by siavash-kadkhodai** [comment]
Hello
It is a great decision to improve GitHub
We support your decision and support you

**Reply by 26bandzz** [reply]
We support what you be doing for use thank you so much

**Reply by Tinz86** [reply]
Amazing and Only the Right thing to do ,Grow!?

**Comment by GAVID1020examplysabah** [comment]
سلام .آیا شما از انجمن GitHub  هستید ؟ من محمد رضا جاوید هستم .حدود اوایل عید نوروز امسال شخصی اکانت گوگل من را هک نمود ودر پی آن شروع به سو استفاده از حساب من در سایت های مختلف کرد .من در همان اوایل به سایت GitHub  در باره ای این سواستفاده گزارش دادم .به هرحال از شش مخزن من با کمک GitHub سه مخزن را از هکر پس گرفتم .اما برای راه اندازی آنها توانای ندارم و تا کنون نتوانسته ام برداشتی از حسابم داشته باشم .خواهش میکنم با این بحران مالی که با آن در این برحه مواجه هستم به من کمک کنید تا این مخازن را راه اندازی خودکار کنم .وکیف پولی را نصب کنم .با سپاس از شما .

**Comment by Comblix** [comment]
![Uploading Screenshot_20220920_183803_com.android.gallery3d.jpg…]()

**Comment by 3kh0** [comment]
I hope to see projects become better and better!

**Comment by kyawzwar** [comment]
liberalkyawzwar@gmail.com

**Comment by yewinag** [comment]
the feature would be great, support you

**Comment by jaskarandji** [comment]
That will help the project to be more understandable .

**Comment by Kve-sudo** [comment]
I have no fucking idea how the fuck to install pycharm on my Chromebook
I've been trying to install it for five days and I've already been driven
crazy by so many beggar commands I've entered. The furthest I've gotten it
says this page or site is available in the Linux terminal. and that pure
fucking letters and doodles.  This mother is harder to penetrate than a
Swiss bank vault.

**Comment by michellemerrill** [comment, marked as answer]
Many thanks for all your responses, we really appreciate your input as we evolve project templates. Thanks again!
